### PR TITLE
document allowlist for github auth

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Creating a new PR to fix conflicts in https://github.com/okteto/docs/pull/469

### CODE REVIEW

File: src/content/reference/helm-chart-values.mdx [hunk 1]
```diff
@@ -151,6 +151,18 @@ auth:
 
 The `organization` field is optional. Only members of the organization will be allowed to log in into your Okteto instance. An empty `organization` field permits any user to log in.
 
+If you want to provide access to only select users of your organization, you can additionally use the `allowList` key to list the GitHub accounts you want to allow.
```
Comment (pchico83) [inline src/content/reference/helm-chart-values.mdx:154]: `allowList` is for any user not belonging to the organization, we cannot filter out members of the organization. To do that, you wouldn't define any organization and list all the allowed users in `allowList`

Comment (omnomagonz) [reply src/content/reference/helm-chart-values.mdx:154]:
```suggestion
You can provide access to select users with the `allowList` key to list the GitHub accounts you want to allow. This feature only works for listing individual GitHub accounts and does not work in conjunction with GitHub organizations.
```

Suggested change based on what I'm understanding here 😄

**Comment by pchico83** [comment]
@RinkiyaKeDad could you update the PR? `allowList` can be used with or without the `organization` key. If there is no `organization`, only these users are allowed to login. Id `organization` is defined, then these users are allowed to login even if they don't belong to your organization. But everyone belonging to the organization is allowed to login

**Comment by codyjlandstrom** [comment]
This PR is getting stale. @RinkiyaKeDad please update, otherwise I'll close and make a ticket next week.

**Comment by RinkiyaKeDad** [comment]
Updated the PR, sorry for the delay!

**Review by pchico83 — APPROVED** [review]

**Pull request merged**